src/bit_vec.rs

```diff
@@ -42,6 +43,19 @@ impl<O: BitOrder, T: BitStore + ToByteSlice> Encode for BitSlice<O, T> {
 	}
 }
 
+fn reverse_endian(vec_u8: &mut Vec<u8>, size_of_t: usize) {
+	for i in 0..vec_u8.len() / size_of_t {
+		for j in 0..size_of_t / 2 {
+			vec_u8.swap(i * size_of_t + j, i * size_of_t + (size_of_t - 1) - j);
+		}
+	}
+}
+
+/// # WARNING
+///
+/// In bitvec v0.17.4 the only implementation of BitStore are u8, u16, u32, u64, usize.
+/// This implementation actually only support u8, u16, u32 and u64, encoding of with BitStore being
+/// usize is inconsistent between plateform.
```

Comment:
hopefully bitvec will never release a new minor version of 0.17 with more implementation of BitStore.

Reply:
but then if user have implemented something generically using `BitStore` trait, then they will have to use our new trait no ?